src/language-css/parser-postcss.js

```diff
@@ -525,7 +525,7 @@ function parseNestedCSS(node, options) {
         return node;
       }
 
-      if (lowercasedName === "import") {
+      if (["import", "use", "forward"].includes(lowercasedName)) {
```

Comment:
Could you define a new function as following:

```js
const moduleRuleNames = new Set(["import", "use", "forward"]);
function isModuleRuleName(name) {
  return moduleRuleNames.has(name);
}
```

And replace this to

```js
if (isModuleRuleName(lowercasedName)) {
```

?

Reply:
Done!